contracts-periphery/.env.exmaple

```diff
@@ -0,0 +1,2 @@
+ETH_RPC_URL=
+PRIVATE_KEY=
```

Comment:
I don't see this used anywhere, what's it for?

Reply:
I think it's remnants from the Makefile we used as a template, which had a private key input for deployment. If this isn't used anywhere right now I agree we should remove it